.chloggen/version-07.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adopting new Datadog Agent API levels
```

```suggestion
note: Add support for Datadog Agent API v0.7 
```
I think calling out the specific version would probably help anyone reading this changelog entry determine that this is relevant to them.